### ISSUE TITLE
How to Access the thresholds in a ‘Linear Mixer Transition’ asset?

### ISSUE THREAD

**Issue by LarryP15** [KybernetikGames/animancer#99]
In the ‘03 Linear Blending’ example, there is a 

[SerializeField] private LinearMixerTransition _Mixer; 

accessible to the code.  In this example, there are three Animation Clips, Humanoid-Idle, Humanoid-Walk, and Humanoid-Run, each with its own threshold number.

In my game I will have over a hundred characters and they will all have somewhat different thresholds associated with the animation.  I don’t want to move them much faster than their fastest threshold number so I would like to access the max threshold from _Mixer.

So basically, I would like to get the threshold number for each animation clip in the LinearMixerTransition asset.

How would I go about doing that? (I’m a somewhat inexperienced programmer).  

Any tips 

Thanks,

Larry

**Comment by KybernetikGames** [comment]
`_Mixer.Transition.Thresholds`

**Comment by LarryP15** [comment]
Thanks!
